{
  "debianbug": 859494,
  "description": "Incorrect interaction of the parse_packet() and parse_part_sign_sha256() functions in network.c in collectd 5.7.1 and earlier allows remote attackers to cause a denial of service (infinite loop) of a collectd instance (configured with \"SecurityLevel None\" and with empty \"AuthFile\" options) via a crafted UDP packet.",
  "releases": {
    "buster": {
      "fixed_version": "5.7.2-1",
      "repositories": {
        "buster": "5.8.1-1.3"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "nodsa": "Minor issue",
      "nodsa_reason": "",
      "repositories": {
        "jessie": "5.4.1-6+deb8u1",
        "jessie-security": "5.4.1-6+deb8u1"
      },
      "status": "open",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "5.7.2-1",
      "repositories": {
        "sid": "5.8.1-1.3"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "nodsa": "Minor issue",
      "nodsa_reason": "",
      "repositories": {
        "stretch": "5.7.1-1.1"
      },
      "status": "open",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}